src/app/components/Home/Office/Canvas/PopUpInfoPlace/index.tsx

```diff
@@ -197,52 +191,60 @@ const PopUpInfoPlace: React.FC<IPopUpInfoPlace> = ({
       <Dialog
         onClose={handleEventPopUp}
         aria-labelledby="customized-dialog-title"
-        open={visibility}
+        open={dataAboutWorkplace !== null}
       >
-        <DialogTitle id="customized-dialog-title" onClose={handleEventPopUp}>
-          {dataAboutWorkplace.type} - {dataAboutWorkplace.label}
-        </DialogTitle>
-        <DialogContent dividers>
-          <Typography className={classes.occupantUser}>
-            Occupant: {dataAboutWorkplace.occupant}{" "}
-          </Typography>
-          <form className={classes.container} noValidate>
-            <TextField
-              disabled={dataSelectedWorkplace.isDisabled}
-              id="datetime-booking-place-start"
-              label="Start booking"
-              type="date"
-              defaultValue={dataSelectedWorkplace.startReservation}
-              onChange={handlerChange}
-              className={classes.textField}
-              InputLabelProps={{
-                shrink: true,
-              }}
-            />
-            <TextField
-              disabled={dataSelectedWorkplace.isDisabled}
-              id="datetime-booking-place-end"
-              label="End booking"
-              type="date"
-              defaultValue={dataSelectedWorkplace.endReservation}
-              onChange={handlerChange}
-              className={classes.textField}
-              InputLabelProps={{
-                shrink: true,
-              }}
-            />
-          </form>
-        </DialogContent>
-        <DialogActions>
-          <Button
-            disabled={dataSelectedWorkplace.isDisabled}
-            onClick={handleEventButton}
-            variant="contained"
-            color="primary"
-          >
-            Save changes
-          </Button>
-        </DialogActions>
+        {dataAboutWorkplace && (
+          <>
+            {" "}
```

Comment:
Seems that empty space in code was replaced with `{" "}` during automated code formatting. Please, remove it.